.github/workflows/cron-tasks.yml

```diff
@@ -0,0 +1,49 @@
+on:
+  # Once a week or on pushes to master
+  schedule:
+    - cron: "0 3 * * 0"
+  push:
+    branches:
+      - master
+
+jobs:
+  update_third_party_notices:
+    name: Update THIRD_PARTY_NOTICES
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v2
+      - name: Install Dependencies
+        run: npm run bootstrap-ci
+      - name: Update THIRD_PARTY_NOTICES.md
+        run: npm run update-third-party-notices
+      - name: Commit changes
+        run: |
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
+          git commit --no-allow-empty -m "chore: update THIRD_PARTY_NOTICES" THIRD_PARTY_NOTICES.md || true
+      - name: Push updates
+        uses: ad-m/github-push-action@v0.6.0
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+
+  update_authors:
+    name: Update AUTHORS
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v2
+      - name: Install Dependencies
+        run: npm run bootstrap-ci
+      - name: Update THIRD_PARTY_NOTICES.md
+        run: npm run update-third-party-notices
+      - name: Commit changes
+        run: |
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
+          git add AUTHORS \*/AUTHORS
+          git commit --no-allow-empty -m "chore: update AUTHORS" || true
+      - name: Push updates
+        uses: ad-m/github-push-action@v0.6.0
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think we’d want these in the same jobs, because otherwise the pushes race against each other and only one will come through (we’re already having this problem when we merge 2 PRs right after each other – one run for the new commits on master will fail because the other’s already pushed by the time it’s finished)

Reply:
Ah good point! Did not think of that. I'd then move it in the same job but keep it in separate commits?

Reply:
Yeah, sounds fine to me :)